content/proposals.md

```diff
@@ -194,12 +194,12 @@ is as follows:
         `1234` is the MSC ID.
     -   Your PR description must include a link to the rendered Markdown
         document and a summary of the proposal.
-    -   It is often very helpful to link any related MSCs or [matrix-doc
-        issues](https://github.com/matrix-org/matrix-doc/issues) to give
+    -   It is often very helpful to link any related MSCs or [matrix-spec
+        issues](https://github.com/matrix-org/matrix-spec/issues) to give
         context for the proposal.
     -   Additionally, please be sure to sign off your proposal PR as per
         the guidelines listed on
-        [CONTRIBUTING.rst](https://github.com/matrix-org/matrix-doc/blob/main/CONTRIBUTING.rst).
+        [CONTRIBUTING.rst](https://github.com/matrix-org/matrix-spec/blob/main/CONTRIBUTING.rst).
```

Comment:
we should give matrix-spec-proposals its own CONTRIBUTING.md, but that can happen another day I suppose.